packages/cli/src/__tests__/fixtures/sub-command/node_modules/sub-command/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "sub-command",
   "version": "1.0.0",
-  "main": "index.mjs"
+  "main": "./dist/index.js"
```

Comment:
This is a ne requirement for sub-clis, they must export an `index.js` in a `dist` folder. The `index.js` can be esm or cjs, that does not matter.